api/proto/config/appconfig.proto

```diff
@@ -26,6 +26,47 @@ enum MetaDataType {
   MetaDataDriveMultipart = 3; // Process multipart MIME for application
 }
 
+// The information about a snapshot
+message AvailableSnapshot {
+  string id = 1;
+}
+
+// The snapshot information for an Application Instance
+message SnapshotInfo {
```

Comment:
What about `SnapshotConfig` name instead, to distinguish between pulled config vs pushed info?

Reply:
Yep, maybe it's a good idea. Just to follow our agreements.

Reply:
It's maybe funny but yesterday i wanted to rename it in the design document but then returned it back.